docs/install/install.md

```diff
@@ -0,0 +1,458 @@
+---
+myst:
+  html_meta:
+    "description": "Installation notes for Plone Classic UI"
+    "property=og:description": "Installation notes for Plone Classic UI"
+    "property=og:title": "Installation notes for pip and buildout based installation paths"
+    "keywords": "Plone, Classic UI, classic-ui, installation, pip, buildout"
+---
+
+(classic-ui-installation-label)=
+
+# Installation
+
+This sections describes how to install Plone 6 Classic UI.
+Examples include the following.
+
+- pip based install method
+- buildout based install method
```

Comment:
I would remove the above section and break apart this file into two new pages:

- Install Classic UI via pip
- Install Classic UI via buildout